app/src/main/res/layout/fragment_contact_us.xml

```diff
@@ -0,0 +1,149 @@
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/n_60"
+    tools:context=".contactusform.ContactUsFragment">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/card_view_layout"
+        android:layout_width="0dp"
+        style="@style/default_card_style"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/constraintLayout_bottom"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:minHeight="500dp">
+
+            <ImageView
+                android:id="@+id/imageView_at_sign"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/size_3x"
+                android:layout_marginTop="@dimen/size_3x"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/textView_main"
+                app:srcCompat="@drawable/contact_us_at_sign" />
+
+            <ImageView
+                android:id="@+id/imageView_phone_symbol"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/size_3x"
+                android:layout_marginTop="@dimen/size_2x"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/imageView_at_sign"
+                app:srcCompat="@drawable/contact_us_phone_symbol" />
+
+            <ImageView
+                android:id="@+id/imageView_pin"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/size_3x"
+                android:layout_marginTop="@dimen/size_2x"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/imageView_phone_symbol"
+                app:srcCompat="@drawable/contact_us_pin" />
+
+            <com.safetyheads.akademiaandroida.H2View
+                android:id="@+id/textView_main"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/size_3x"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.0"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:text="@string/contact_us_main_text" />
+
+            <TextView
+                android:id="@+id/textView_email"
```

Comment:
It would be nice if user after click on email will be able to open E-mail client.